catroid/src/main/java/org/catrobat/catroid/content/bricks/BroadcastReceiverBrick.java

```diff
@@ -158,23 +157,19 @@ private void setSpinnerSelection(Spinner spinner) {
 	// TODO: BroadcastBrick and BroadcastReceiverBrick contain this identical method.
 	private void showNewMessageDialog(final Spinner spinner) {
 		final Context context = spinner.getContext();
-		BrickTextDialog editDialog = new BrickTextDialog() {
+		BrickTextDialog editDialog = new BrickTextDialog(R.string.dialog_new_broadcast_message_title, R.string
+				.dialog_new_broadcast_message_name, context.getString(R.string.new_broadcast_message)) {
```

Comment:
Maybe there should be a second constructor at the TextDialog which takes three integer, for a better readability.